### ISSUE TITLE
CI Does Not Install FFMPEG

### ISSUE THREAD

**Issue by imaspacepirate** [UofU-CS3505-S20/Project-1#2]
# What's Wrong
The CI script builds FFMPEG correctly but fails to install it with the following error:
```bash
make -j41 -l40 -C ./ffmpeg install 
make[1]: Entering directory '/home/runner/work/Project-1/Project-1/ffmpeg'
mkdir: cannot create directory ‘./ffmpeg’: Not a directory
mkdir: cannot create directory ‘./ffmpeg’: Not a directory
make[1]: *** [install-html] Error 1
doc/Makefile:117: recipe for target 'install-html' failed
make[1]: *** Waiting for unfinished jobs....
doc/Makefile:125: recipe for target 'install-man' failed
make[1]: *** [install-man] Error 1
make[1]: Leaving directory '/home/runner/work/Project-1/Project-1/ffmpeg'
make: *** [install_ffmpeg] Error 2
Makefile:37: recipe for target 'install_ffmpeg' failed
[error]Process completed with exit code 2.
```
# Affected Files
./Makfile

# Recommendations
1. Investigate ./Makefile.

**Comment by imaspacepirate** [comment]
Fixed by commit 3e2bfc9.

**Comment by imaspacepirate** [comment]
Fixed by commit 3e2bfc9.

**Comment by imaspacepirate** [comment]
Fixed by commit 3e2bfc9.
